spot-client/src/common/app-state/setup/selectors.js

```diff
@@ -30,3 +31,24 @@ export function getMeetingOptions(state) {
         showMeetingToolbar: state.setup.showMeetingToolbar
     };
 }
+
+/**
+ * A selector which returns whether or not the application has been configured.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {boolean}
+ */
+export function isSetupComplete(state) {
+    return state.setup.completed;
+}
+
+/**
+* A selector which returns whether or not the current client is a Spot
```

Comment:
This is a bit confusing when Im reading it knowing about Spot-TV and Spot-Remote. Is it about Spot-TV ? Is it the flag that's read from presence ?

Reply:
This method was there already is just moved around, right ?

Reply:
Yeah, I moved it; didn't modify its contents.